node/defaults.go

```diff
@@ -57,11 +57,11 @@ func DefaultDataDir() string {
 	home := homeDir()
 	if home != "" {
 		if runtime.GOOS == "darwin" {
-			return filepath.Join(home, "Library", "Ethereum")
+			return filepath.Join(home, "Library", "Callisto")
 		} else if runtime.GOOS == "windows" {
-			return filepath.Join(home, "AppData", "Roaming", "Ethereum")
+			return filepath.Join(home, "AppData", "Roaming", "Callisto")
 		} else {
-			return filepath.Join(home, ".ethereum")
+			return filepath.Join(home, ".callisto")
```

Comment:
will it work with multi-geth without issues?

Reply:
@yograterol Nope if you are going to migrate from CLO/1.1 to multi-geth you need to move default filepath as I've said already.